config.yaml

```diff
@@ -144,64 +146,84 @@ params:
+          - name: "CONP"
+            image: /images/logos/conp_logo.png
+            url: https://conp.ca/
+      - name: Hackathon supporters
+        oursponsors:
+          - name: "WIN"
+            image: /images/logos/logo_WIN_clear.png
+            url: https://www.win.ox.ac.uk/
+          - name: "UNIQUE"
+            image: /images/logos/Unique_logo.png
+            url: http://unique.quebec
+          - name: "Quebec Bio-Imaging Network"
+            image: /images/logos/QBIN_logo.png
+            url: https://www.rbiq-qbin.qc.ca/Accueil
+      - name: Sponsors
+        oursponsors:
+          - name: "OHBM Australian chapter"
+            image: "/images/logos/OHBM_australian_chapter.png"
+            url: "https://ohbmaustralia.wordpress.com/"
 
   section2:
     title: Open science is an umbrella.
     subtitle: We're here to help grow the community supporting it.
     id: openumbrella
     features:
-    - title: Open data
-      text: Sharing neuroimaging data isn't easy, but we can make it easier. We're interested in understanding how OHBM members share data, where they encounter difficulties, and what kinds of new infrastruce and ethics can make next-generation data-sharing possible.
-      # Icon (from /images/illustrations/icons/___.svg)
-      icon: mouse-globe
-    - title: Open code
-      text: Analysis code is often a second-class citizen in neuroimaging research, but we can change that. We help to teach scientists best-practices in software development—and how those practices can result in real-world gains in research quality and output.
-      icon: laptop-cloud
-    - title: Open publishing
-      text: Papers shouldn't hide behind paywalls. We want to help create solutions to enable new, open-acess platforms and to facilitate free information.
-      icon: doc-sync
+      - title: Open data
+        text: Sharing neuroimaging data isn't easy, but we can make it easier. We're interested in understanding how OHBM members share data, where they encounter difficulties, and what kinds of new infrastruce and ethics can make next-generation data-sharing possible.
+        # Icon (from /images/illustrations/icons/___.svg)
+        icon: mouse-globe
+      - title: Open code
+        text: Analysis code is often a second-class citizen in neuroimaging research, but we can change that. We help to teach scientists best-practices in software development—and how those practices can result in real-world gains in research quality and output.
+        icon: laptop-cloud
+      - title: Open publishing
+        text: Papers shouldn't hide behind paywalls. We want to help create solutions to enable new, open-acess platforms and to facilitate free information.
+        icon: doc-sync
+
+  elections:
+    title: Join the OSSIG
+    subtitle: We are excited to announce this year's Nominations for OHBM 2021!
+    id: openumbrella
+    features:
+      - title: Insert here motivational text why you should join the OSSIG
```

Comment:
needs updating

Reply:
Hi @Remi-Gau! Did you want me to suggest text for things like `Insert here motivational text why you should join the OSSIG`? I can do it in suggestions or my own PR, which would you prefer?

Reply:
I am not fussy. Whichever is more convenient for you. :-)

Reply:
I'll just reply here, then!

Reply:
`Insert here motivational text why you should join the OSSIG`:

If you are excited by open science, have enjoyed and benefitted from Hackathon and/or Open Science Room activity in your local hubs and globally, or have ideas about how we can improve on OHBM's open science activities in the future, please consider joining us! By self-nominating for a role as an Open Science SIG officer, you can join our passionate and enthusiastic team and help push for more reproducibility, transparency, and openness in science, as well as inclusivity and openness in our community. Early career researchers, BIPOC, LGBTQ+, and people from outside North America and Europe are especially encouraged to apply!

(any edits are 100% welcome)

Reply:
> By self-nominating for a role as an Open Science SIG officer, you can join our passionate and enthusiastic team and help push for more reproducibility, transparency, and openness in science, as well as inclusivity and openness in our community.

Edit from Cass: By self-nominating for a role as an Open Science SIG officer, you can join our passionate and enthusiastic team and help push for more reproducibility, transparency, and openness in science, as well as *improved inclusivity and access in our community.*